### PR TITLE
Prepare February Release

### PR DESCRIPTION
**Pull request by 0snap**
###  :notebook_with_decorative_cover: Description

- Pin versions for different plugins to keep compatibility with older versions installed on PyPI
- Disable `dev-mode` Make target for un-migrated plugins (resp. STIX-2)

This PR is branched off https://github.com/tenzir/threatbus/pull/99 - that has to be merged first.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/threatbus](https://docs.tenzir.com/threatbus), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

File-by-file.

### CODE REVIEW

**Comment by dominiklohmann** [comment]
> This PR is branched off #99 - that has to be merged first.

Side note: You can set the merge branch of this PR accordingly to make it independent of the merge order. You can change this when editing the title of the PR.

**Review by dominiklohmann — COMMENTED** [review]
For VAST, we recently changed the CHANGELOG format slightly. Do you want to make the same adaptions here?

**Comment by 0snap** [comment]
@dominiklohmann No, I like the format as it is right now. There are not so many changes that I see an urge to group by type. Also, it reads nicely in chronological order.

File: Makefile [hunk 1]
```diff
@@ -85,10 +85,10 @@ install:
...
 dev-mode:
 	pip install --editable .
-	$(MAKE) -C plugins/apps/threatbus_zeek dev-mode
-	$(MAKE) -C plugins/apps/threatbus_misp dev-mode
 	$(MAKE) -C plugins/apps/threatbus_zmq_app dev-mode
-	$(MAKE) -C plugins/apps/threatbus_cif3 dev-mode
 	$(MAKE) -C plugins/backbones/threatbus_inmem dev-mode
 	$(MAKE) -C plugins/backbones/threatbus_rabbitmq dev-mode
-	$(MAKE) -C apps/vast dev-mode
+	# $(MAKE) -C plugins/apps/threatbus_zeek dev-mode
```
Comment (lava) [inline Makefile:91]: Is this a left-over? If not, I'd suggest adding a comment to explain why these are left here but commented out.

**Review by lava — APPROVED** [review]
Looks good to me module one minor nit.

**Pull request merged**